Bowling_Calculator/BowlingCalculator.cs

```diff
@@ -0,0 +1,283 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace BowlingCalculator
```

Comment:
It is better if namespace name is equal to folder name
```suggestion
namespace Bowling_Calculator
```